lib/clashom.gi

```diff
@@ -147,22 +147,22 @@ local clT,	# classes T
       localcent_r, # image
       cr,
       isdirprod,# is just M a direct product
-      genpos,	# generator index
+      genpos,        # generator index
       genpos2,
-      gen,	# generator
-      stab,	# stabilizer
-      stgen,	# local stabilizer generators
+      gen,        # generator
+      stab,        # stabilizer
+      stgen,        # local stabilizer generators
```

Comment:
more comment indentation weirdness. Could it be that the script used to convert tabs to spaces blindly replaces each tab by 8 spaces, instead of aligning to the next multiple-of-8 column?

I am using the standard Unix util `expand` (following https://stackoverflow.com/a/11094620/928031 but adjusted to use tab width 8, i.e. removing the `-t 4`), and it seems to get these cases right.

Reply:
Thanks @fingolfin, I am doing it by hand, so I'm to blame, I think I've fixed that now, I'll take a look at `expand`, sounds more efficient!

Reply:
Thanks James!